sdk/appconfiguration/Azure.ResourceManager.AppConfiguration/src/Azure.ResourceManager.AppConfiguration.csproj

```diff
@@ -7,7 +7,7 @@
       This is a beta preview vesion. This version uses a next-generation code generator that introduces important breaking changes, but also new features (such as intuitive authentication, custom HTTP pipeline, distributed tracing and much more).
     </Description>
     <PackageTags>azure;management;appconfiguration</PackageTags>
-    <NoWarn>$(NoWarn);AZC0001;AZC0008;</NoWarn>
+    <NoWarn>$(NoWarn);AZC0008;</NoWarn>
```

Comment:
I assume this is still planning to be addressed in a future update.

Reply:
yeah, this is just to get rid of 0001, the others will follow